client/src/components/styleThumbnails.jsx

```diff
@@ -1,47 +1,54 @@
-// style thumbnails go here
-// How will you display the style images?
-// Will you use the thumbnail or the full image from the style?
-// How will a user see they selected a style?
-// What will the badge look like?
-// How will you grab the image url and the style id?
-// What will be held in state?
-import React, { useCallback, useState } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import Container from 'react-bootstrap/Container';
-// import Row from 'react-bootstrap/Row';
-// import Col from 'react-bootstrap/Col';
-// import Card from 'react-bootstrap/Card';
 import Image from 'react-bootstrap/Image';
 import { v4 as uuidv4 } from 'uuid';
 
-const StyleThumbnails = ({ styles }) => {
-  const [isSelected, imgIsSelected] = useState(false);
-  const photoArr = styles.results[0].photos;
-  const handleStyleSelect = useCallback(() => {
-    imgIsSelected(!isSelected); // current style will go here
-  }, [isSelected]);
+const StyleThumbnails = ({ styles, setCurrentStyle, currentStyle }) => {
+  const stylesArr = Object.entries(styles.results);
 
-  let className = 'thumbnails-img ';
-  if (isSelected) {
-    className += 'thumbnails-selected';
-  }
+  const handleStyleSelect = (styleId) => {
```

Comment:
className might be a bit too vague.. Totally up to you tho!

Reply:
It is the classname for the className in the Image so I tried to just keep everything the same, but would love any recommendations for renaming! :D